app/Exceptions/Handler.php

```diff
@@ -36,6 +37,20 @@ class Handler extends ExceptionHandler
         'password_confirmation',
     ];
 
+    public function render($request, Throwable $exception)
+    {
+        if ($this->isHttpException($exception)) {
+            $status = $exception->getStatusCode();
+            $view = "errors.{$status}";
+
+            if (View::exists($view)) {
+                return response()->view($view, ['exception' => $exception], $status);
+            }
+        }
+
+        return parent::render($request, $exception);
+    }
+
     /**
```

Comment:
Why is this needed, doesn't Laravel automatically handle the rendering of error pages?